src/custom/components/ScrollToTop/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { ArrowUpCircle } from 'react-feather'
+import useDebouncedChangeHandler from 'utils/useDebouncedChangeHandler'
+
+interface WrapperProps {
+  top?: string
+  right?: string
+  bottom?: string
+  left?: string
+  background?: string
+}
+
+const Wrapper = styled.div<WrapperProps>`
+    position: fixed;
+    ${({ top }): string | undefined => top && `top: ${top};`}
+    ${({ left }): string | undefined => left && `left: ${left};`}
+    ${({ bottom }): string | undefined => bottom && `bottom: ${bottom};`}
+    ${({ right }): string | undefined => right && `right: ${right};`}
+
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center; 
+    align-items: center;
+
+    font-size: larger;
+    background: ${({ background, theme }) => background || theme.primary1}
+    border-radius: ${({ theme }) => theme.buttonPrimary.borderRadius}
+
+    padding: 0.2rem 0.4rem;
+
+    cursor: pointer;
+    opacity: 0.4;
+
+    &:hover {
+        opacity: 1;
+    }
+
+    > svg {
+        margin: 0 0.4rem;
+    }
+
+    transition: opacity 0.4s ease-in-out;
```

Comment:
Nit: why not move it right after line 33? The way I look at the code is in this order:
- Properties for this current element
- Any pseudo elements (&:hover, etc.) still part of this property.
- Target any children

Reply:
:shrug: i can do that